### ISSUE TITLE
nx 11 is not supported anymore

### ISSUE THREAD

**Issue by cakeinpanic** [jscutlery/semver#662]
hi, [this commit](https://github.com/jscutlery/semver/commit/90b9ce90b0e75f72e2ac2e2da7a22f66977ea050) broke compatibility with nx11 which we are extensively using in my company

So now we can not use new features that being added to this project, for example, [this one](https://github.com/jscutlery/semver/pull/659)

Can you please bring compatibility back?

**Comment by bgutschke** [comment]
Breaks on Nx v15.1.1 as well. Same issue. 

Downgrading to `@jscutlery/semver@2.29.3` fixed the issue.

I think it might be possible that the bug is actually in `@nrwl/devkit` and will be fixed in `v15.7.0`. 
see: https://github.com/nrwl/nx/pull/14854

Only had a quick glance, so not 100% sure.

**Comment by cakeinpanic** [comment]
yes, `@jscutlery/semver@2.29.3` also helped me, but I really need that feature I added in a PR and which is released in later versions

**Comment by edbzn** [comment]
Too many breaking changes in this devkit, it's not following semver like the rest of their packages. I discussed this last time and concluded that we should bump a major in @jscutlery/semver for each @nrwl/devkit minor...

**Comment by cakeinpanic** [comment]
@edbzn can you for now bring back compatibility with both devkits?

**Comment by edbzn** [comment]
@cakeinpanic I do not have enough time to give today, unfortunately, and I won't be available for a couple of days, so if you raise a PR I'll merge and release but can't do it myself.
